src/main/java/kr/codesqaud/cafe/controller/UserController.java

```diff
@@ -44,16 +47,21 @@ public String list(Model model) {
     @GetMapping("/profile/{userId}")
     public String profile(@PathVariable String userId,
                           Model model) {
-        User user = userRepository.findUser(userId);
-        model.addAttribute("user", user);
+        Optional<User> optionalUser = userRepository.findUserById(userId);
+        if (optionalUser.isEmpty()) return "fail-something";
```

Comment:
```suggestion
        if (optionalUser.isEmpty()) return "fail-something";
```
이 부분 별로 좋지 않은 것 같아요. fail-something 이 무엇 때문에 발생한지도 모르고 네이밍도 좋지 않고 하는 일도 너무 많아 보입니다.

예외 처리 방법을 학습하고 글로벌 에러 페이지로 보내는 게 좋지 않을까 생각이 드네요.